### PR TITLE
add r-fpp3

### PR DESCRIPTION
**Pull request by cnrrobertson**
Used [r skeleton helper script](https://github.com/bgruening/conda_r_skeleton_helper)

### CODE REVIEW

**Comment by cnrrobertson** [comment]
@conda-forge/help-r, ready for review!

File: recipes/r-fpp3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    - r-tsibbledata >=0.2.0
+
+test:
+  commands:
+    - $R -e "library('fpp3')"           # [not win]
+    - "\"%R%\" -e \"library('fpp3')\""  # [win]
+
+about:
+  home: https://pkg.robjhyndman.com/fpp3package/, https://github.com/robjhyndman/fpp3package,
+    https://OTexts.com/fpp3/
```
Comment (mfansler) [inline recipes/r-fpp3/meta.yaml:67]: Split URLs:

```suggestion
  home: https://OTexts.com/fpp3/
  dev_url: https://github.com/robjhyndman/fpp3package
  doc_url: https://pkg.robjhyndman.com/fpp3package/
```

File: recipes/r-fpp3/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  license: GPL-3
```
Comment (mfansler) [inline recipes/r-fpp3/meta.yaml:68]: SPDX

```suggestion
  license: GPL-3.0-only
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for adding this! ❤️ 

Couple touch ups, but otherwise looks good.

**Review by mfansler — APPROVED** [review]

**Pull request merged**